people/ynahmany.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "יוחאי נחמני",
+  "github": "ynahmany",
+  "facebook": "https://www.facebook.com/yohay.nahmany/",
+  "twitter": "https://twitter.com/ynahmany",
+  "linkedin": "https://www.linkedin.com/in/yohay-nahmany/",
+  "stackoverflow":"https://stackoverflow.com/users/9135929/ynahmany",
+  "description": "מפתח פול סטאק בעשור האחרון, אוהב front end, משימות infrastructure, סטאר וורס ומקלדות מכניות מרעישות במיוחד"
```

Comment:
Please add a color.

Reply:
@MichalPorag change the social to object with url and color?